### PR TITLE
US108138 Menu ui cleanup

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**

### CODE REVIEW

File: d2l-sequence-viewer.js [hunk 1]
```diff
@@ -404,14 +417,20 @@ class D2LSequenceViewer extends mixinBehaviors([
 	}
 
 	_sideBarOpen() {
+		console.log(`offsetWidth: ${this.offsetWidth}`);
```
Comment (pbrown-d2l) [inline d2l-sequence-viewer.js]: This should get removed

File: d2l-sequence-viewer.js [hunk 2]
```diff
@@ -404,14 +417,20 @@ class D2LSequenceViewer extends mixinBehaviors([
...
+		const maxWidth = 1173;
+		const sidebarWidth = Math.round((this.offsetWidth <= maxWidth ? this.offsetWidth : maxWidth) / 3);
 		const offsetWidth = this.$$('#sidebar-occlude').offsetWidth;
-		const marginLeft = String(376 + offsetWidth) + 'px';
+		const marginLeft = String(sidebarWidth + offsetWidth) + 'px';
+		console.log(`sidebarWidth: ${sidebarWidth}`);
```
Comment (pbrown-d2l) [inline d2l-sequence-viewer.js]: this too

File: d2l-sequence-viewer.js [hunk 3]
```diff
@@ -404,14 +418,18 @@ class D2LSequenceViewer extends mixinBehaviors([
 	}
 
 	_sideBarOpen() {
+		const maxWidth = 1170;
+		const sidebarWidth = Math.round((this.offsetWidth <= maxWidth ? this.offsetWidth : maxWidth) / 3);
 		const offsetWidth = this.$$('#sidebar-occlude').offsetWidth;
-		const marginLeft = String(376 + offsetWidth) + 'px';
+		const marginLeft = String(sidebarWidth + offsetWidth) + 'px';
```
Comment (dbatiste) [inline d2l-sequence-viewer.js]: This could just be : 
```
const marginLeft = `${sidebarWidth + offsetWidth}px`;
```

Comment (dbatiste) [reply d2l-sequence-viewer.js]: There's a few other places this could be done, although I it looks like they were probably there already.

**Pull request merged**